### ISSUE TITLE
rpmem: missing info about ulimit settings

### ISSUE THREAD

**Issue by kkajrewicz** [pmem/issues#379]
Remote pool creation might be unsuccessful due to limitations of virtual memory. The reason of failure(errno, error msg) may not be obvious. Those limit may be changed with ulimit tool. Add note to rpmem docs about that.

**Comment by kkajrewicz** [comment]
I remember following problem related to max size that can be locked into memory:
An attempt to create remote pool via rpmem_create() failed with dbg. msg. that pointed to problems with finding verbs provider. When I ran fi_info and grepped against verbs I got nothing. It turned out that due to insufficient max locked-in-memory address space verbs provider was not registered properly.    Indeed, raising limits for 'memlock' to unlimited helped. 
To be honest, clarification added to man page would not help me to solve this problem, I would not link it with nature of my problem
